packages/forms/test/template_integration_spec.ts

```diff
@@ -1821,7 +1821,61 @@ import {NgModelCustomComp, NgModelCustomWrapper} from './value_accessor_integrat
            expect(maxValidateFnSpy).not.toHaveBeenCalled();
            expect(minValidateFnSpy).not.toHaveBeenCalled();
          }));
+      it('should not include the min and max validators for null',
+           fakeAsync(() => {
+                      @Component({template: '<input type="input" min="null" max="null">'})
```

Comment:
I think that this is binding the string `"null"` rather than the value `null`. Should it not be

```suggestion
                      @Component({template: '<input type="input" [min]="null" [max]="null">'})
```